cl/search/views.py

```diff
@@ -659,6 +690,25 @@ def do_es_search(
                 search_form.cleaned_data,
                 courts,
             )
+            cited_cluster = add_depth_counts(
+                # Also returns cited cluster if found
+                search_data=cd,
+                search_results=paged_results,
+            )
+
+            if cd["type"] in [
+                SEARCH_TYPES.OPINION,
+                SEARCH_TYPES.RECAP,
```

Comment:
Why RECAP and Dockets are included here?

Reply:
The `query_citation` object is used within the `search.html` template. Upon examining the original `do_search` method, I noticed that the object is computed for **opinion**, **recap**, and **docket** searches.

https://github.com/freelawproject/courtlistener/blob/9646e5aaf90306d378fd65f106e8c3bf1a13f4a7/cl/search/templates/search.html#L134-L141

https://github.com/freelawproject/courtlistener/blob/9646e5aaf90306d378fd65f106e8c3bf1a13f4a7/cl/search/views.py#L206-L213

Reply:
got it!